docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+
+1. Install `docker` - on Fedora you can use:
+
+   ```
+   sudo dnf install docker
+   ```
+
+   Add yourself to the `docker` group to allow running docker as root:
+
+   ```
+   sudo usermod -aG docker $USER && newgrp docker
+   ```
+
+   Restart docker service to fix the permissions on the docker daemon
+   socket:
+
+   ```
+   sudo systemctl restart docker
+   ```
+
+1. Install `podman` - on Fedora you can use:
+
+   ```
+   sudo dnf install podman
+   ``` 
+   
+### Testing that drenv is healthy
+
+Run this script to make sure `drenv` works. 
+
+   ```
+   cd ramen/test
+   scripts/drenv-selftest
+   ```
+   
+   Example output:
+   ```
+   1. Activating the ramen virtual environment ...
+
+
+   2. Creating a test cluster ...
+
+   2023-11-02 02:50:15,685 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509test] Starting environment
+   2023-11-02 02:50:17,549 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509cluster] Starting minikube cluster
+   2023-11-02 02:50:53,175 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509cluster] Cluster started in 35.63 seconds
+   [...]
+   2023-11-02 02:51:11,359 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509test] Environment started in 55.67 seconds
+
+   3. Deleting the test cluster ...
+
+   [...]
+
+   drenv is set up properly
+   ```
+
+## Starting the test environment
+
+Before using the `drenv` tool to start a test environment, you need to activate the python virtual environment:
+
+   ```
+   cd ramen
+   source venv
+   ```
+
+This environment simulates a Regional DR setup, when the managed clusters are located in different zones (e.g. US, Europe) and storage is replicated asynchronously. This environment includes 3 clusters - a hub cluster (`hub`), and 2 managed clusters (`dr1`, `dr2`) using `Ceph` storage. The connectivity between managed clusters will be configured using subctl (Submariner client). In addition, rbd-mirroring is configured using the Submariner connection for ceph image replication of block volumes.
+
+To start the environment, run:
+
+   ```
+   cd ramen/test
+   drenv start envs/regional-dr.yaml
+   ```
+
+Starting takes 10-15 minutes, depending on your machine and
+internet connection.
+
+Example output:
+   ```
+   2023-11-02 05:26:41,729 INFO    [rdr] Starting environment
+   2023-11-02 05:26:41,860 INFO    [dr1] Starting minikube cluster
+   2023-11-02 05:26:42,948 INFO    [dr2] Starting minikube cluster
+   2023-11-02 05:26:43,929 INFO    [hub] Starting minikube cluster
+   2023-11-02 05:28:23,677 INFO    [dr1] Cluster started in 101.82 seconds
+   [...]
+   2023-11-02 05:40:40,777 INFO    [rdr] Dumping ramen e2e config to '/home/aclewett/.config/drenv/rdr'
+   2023-11-02 05:40:41,129 INFO    [rdr] Environment started in 839.40 seconds
+   ```
+The kubconfigs for dr1, dr2 and hub can be found in $HOME/.config/drenv/rdrkubeconfigs/.
+
+
+Validate that the clusters are created.
+
+   ```
+   minikube profile list
+   ```
+   
+   Example output:   
```

Comment:
Indent left

Reply:
Not clear what to fix

Reply:
The text should not be indented, it may be incorrectly.